### PR TITLE
test: [M3-7321] - Combine billing cypress tests

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
Combines the billing contact Cypress tests into a single test.

## Changes  🔄
List any change relevant to the reviewer.
- The combined tests mock the user's account data and confirm that it is displayed correctly upon page load, then the billing contact information is update, and finally the updates in the information are confirmed to be reflected in the page.
- Saves one test.

## Preview 📷
Screenshot 2023-11-30 at 10 29 13 AM

## How to test 🧪
- Run the following command on your local machine:
 ```
yarn cy:run -s "cypress/e2e/core/billing/billing-contact.spec.ts"
```
- Utilize the cypress interface by running `yarn cy:debug` and then running the `billing-contact.spect.ts` test.

### Verification steps 
- Verify that the test passes on your local machine by uses either of the methods mentioned above.

## As an Author I have considered 🤔
- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]
Nice, thanks @carrillo-erik!

**Review by mjac0bs — COMMENTED** [review]
Thanks for the consolidation. 🧪  CI looks good; I didn't check out locally yet for the reason below.

Before I give this another pass, could you please:
1. Update the PR description to include testing instructions. (Including the command for the test to run, though simple, saves your reviews time!)
2. Add a changeset

Interestingly, I tried to pull this locally with GitHub CLI, as usual, (`gh pr checkout 9940`) and received the following error. I will check out without the CLI command if this persists, but I haven't run into this before and was curious whether you'd seen any issues on your end with the branch.
image

**Comment by carrillo-erik** [comment]
> 1. Update the PR description to include testing instructions.
The PR description has been updated, including the command to run the test locally.

> 2. Add a changeset
I've added the changeset

> Interestingly, I tried to pull this locally with GitHub CLI, as usual, (`gh pr checkout 9940`) and received the following error. I will check out without the CLI command if this persists, but I haven't run into this before and was curious whether you'd seen any issues on your end with the branch.

No, I didn't see any issues.

File: packages/manager/.changeset/pr-9940-tests-1701368286376.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Tests
+---
+
+Combine billing cypress tests ([#9940](https://github.com/linode/manager/pull/9940))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-9940-tests-1701368286376.md:5]:
```suggestion
Combine billing contact Cypress tests ([#9940](https://github.com/linode/manager/pull/9940))
```
Nit: We also have `billing-invoices.spec.ts` and `smoke-billing-activity.ts`, so more specific would be best.

**Review by mjac0bs — APPROVED** [review]
Thanks for updating the PR description for completeness, looks great!

image
Test passed locally and code changes and comments look good. Left one very minor nitpick on the changeset description. 🚢 

(Still not sure what was happening with the GitHub CLI command when checking this out -- that was weird -- but I set up the remote, fetched your branches, and then checked this one out and that was fine.)

**Pull request merged**